### PR TITLE
Add Michael Houmann's solution

### PR DESCRIPTION
**Pull request by mhoumann87**
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

File: web_development_101/javascript_basics/project_rock_paper_scissors.md [hunk 1]
```diff
@@ -11,28 +12,30 @@ We're going to make a simple implementation of grade-school classic "rock paper
 4. Write a function that plays a single round of Rock Paper Scissors.  The function should take two parameters - the `playerSelection` and `computerSelection` - and then return a string that declares the winner of the round like so: `"You Lose! Paper beats Rock"`
    1. make your function case insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation)
 
-   2. __Important note:__ you want to `return` the results of this function call, _not_ `console.log()` them.  To test this function console.log the results:
+2. **Important note:** you want to `return` the results of this function call, _not_ `console.log()` them. To test this function console.log the results:
+
+   ```javascript
```
Comment (KevinMulhern) [inline web_development_101/javascript_basics/project_rock_paper_scissors.md]: This needs to use `~~~` as the delimiters for code blocks due to the markdown parser we use on the main site. Can you change this back please?

File: web_development_101/javascript_basics/project_rock_paper_scissors.md [hunk 2]
```diff
@@ -71,7 +74,7 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 - [Eliezer's Console Solution](https://github.com/vzla0094/rock_paper_scissors/blob/master/practica.js)
 - [Jieselle's Console Solution](https://github.com/jieselle/Rock-Paper-Scissors-Game)
 - [Allan Glasier's Solution](http://allanglasier.com/rock-paper-scissors/) - [View in Browser](http://allanglasier.com/rock-paper-scissors/)
-- [Allan Glasier's Console Solution](https://github.com/allan-glasier/rock-paper-scissors/blob/master/index(original).html)
+- [Allan Glasier's Console Solution](<https://github.com/allan-glasier/rock-paper-scissors/blob/master/index(original).html>)
```
Comment (KevinMulhern) [inline web_development_101/javascript_basics/project_rock_paper_scissors.md]: why were angle brackets added here?

**Comment by mhoumann87** [comment]
The issues should be solved now

**Comment by CouchofTomato** [comment]
Hi @mhoumann87 

There seem to have been a lot of changes made to 2 files, a total difference of 5 changes, mostly seem related to whitespace.

Is there a reason for that?

**Comment by mhoumann87** [comment]
I am quite baffled about what is going on here. In the first pull request I only added my solution and name and nothing else. In the second pr I changed what you mentioned in your comments, and nothing else, and now there are more changes happening? You can just delete my pull request. I am done with this now. Just forget all about me.

**Comment by CouchofTomato** [comment]
Hi @mhoumann87 

I'm sorry you've had problems with it. If you view the file changes you'll see there were a lot of changes made https://github.com/TheOdinProject/curriculum/pull/9463

This makes it very hard to accept.

Wishing you best of luck with your future endeavours

**Pull request closed without merging**